.vib/mediawiki/goss/goss.yaml

```diff
@@ -1,28 +1,35 @@
 file:
-  /opt/bitnami/mediawiki/cache:
+  /bitnami/mediawiki:
     exists: true
     filetype: directory
-    mode: "0775"
-    owner: daemon
-    group: root
-  /opt/bitnami/mediawiki/images:
-    filetype: symlink
-    linked-to: /bitnami/mediawiki/images
+    mode: "2775"
+    owner: root
+  /etc/hosts:
     exists: true
-  /bitnami/mediawiki/LocalSettings.php:
-    mode: "0644"
     filetype: file
-    exists: true
-# Checks the user running the container is non-root
+    contains:
+    {{ range .Vars.hostAliases }}
+    {{ $ip := .ip }}
+    {{ range $host := .hostnames }}
+      - /{{ $ip }}.*{{ $host }}/
+    {{ end }}
+    {{ end }}
+http:
+  https://mediawiki:{{ .Vars.service.ports.https }}:
+    status: 301
+    no-follow-redirects: true
+    allow-insecure: true
 command:
-  user-id-test:
-    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+  check-user-info:
+    exec: id
     exit-status: 0
-    stdout: []
-    stderr: []
-  which-php:
-    exec: which php
+    stdout:
+      - uid={{ .Vars.containerSecurityContext.runAsUser }}
+      - /groups=.*{{ .Vars.podSecurityContext.fsGroup }}/
+  check-db-credentials:
+    exec: env
     exit-status: 0
     stdout:
-    - "/opt/bitnami/php/bin/php"
-    stderr: []
+      - /{{ .Vars.mariadb.auth.database }}/
+      - /{{ .Vars.mariadb.auth.username }}/
+      - /{{ .Vars.mariadb.auth.password }}/
```

Comment:
This is inaccurate, the values could be set in the wrong env variables and the application would still work. Instead, use the following pattern:

```
- /NAME_OF_VAR.*{{ .Vars.PATH.TO.VALUE }}/
```